src/fragments/lib/graphqlapi/ios/subscribe-data.mdx

```diff
@@ -4,18 +4,18 @@ Because the lifetime of the subscription will last longer than the lifetime of a
 
 <BlockSwitcher>
 
-<Block name="Listener (iOS 11+)">
+<Block name="Async/Await">
 
 ```swift
-var subscription: GraphQLSubscriptionOperation<Todo>?
+var subscription: AmplifyAsyncThrowingSequence<GraphQLSubscriptionEvent<Todo>>
 ```
 
 </Block>
 
 <Block name="Combine">
 
 ```swift
-var subscription: GraphQLSubscriptionOperation<Todo>?
+var subscription: AmplifyAsyncThrowingSequence<GraphQLSubscriptionEvent<Todo>>
 var dataSink: AnyCancellable?
```

Comment:
on a high level, i don't think we need the sequence anymore, only need the AnyCancellable, since that will hold onto the sequence reference. Just having this i think is enough:
```
var subscription: AnyCancellable?
```

The calling code below can create a local sequence variable, and pass that into the Amplify.Publisher.create which returns the AnyCancellable to be held by this property here